app/views/proposals/show.html.erb

```diff
@@ -185,11 +185,7 @@
 <% if policy(@proposal).can_approve_or_reject? %>
   <div class="centered">
     <p>
-      <%= link_to "Approve",
-          approval_response_path(approver_action: 'approve',
-                                 # @todo: get rid of cart
-                                 cart_id: @proposal.cart.id, version: @proposal.version
-                                 ), class: 'form-button' %>
+      <%= link_to "Approve", approve_proposal_url(@proposal, version: @proposal.version), class: 'form-button' %>
```

Comment:
It's added in `approval_action_url` within the mailer helper...not sure if we should combine them, since that needs a URL not a path, and this one shouldn't have a token. Thoughts?